util/monoclock.h

```diff
@@ -19,4 +19,10 @@ int monoclock_get(struct timeval *);
  */
 int monoclock_get_cputime(struct timeval *);
 
+/**
+ * monoclock_diff_seconds(start, stop):
+ * Calculate the difference (in seconds) between a start time and an end time.
+ */
+long double monoclock_diff_seconds(struct timeval, struct timeval);
```

Comment:
How about making this a macro?
```
#define timeval_diff(x, y) ((y.tv_sec - x.tv_sec) + (y.tv_usex - x.tv_usec) * 0.000001)
```

I'd be inclined to include it in `monoclock.h` but call it `timeval_diff` since it applies to timevals in general.